yt_dlp/extractor/piaulizaportal.py

```diff
@@ -0,0 +1,77 @@
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    int_or_none,
+    parse_qs,
+    time_seconds,
+    traverse_obj,
+)
+
+
+class PIAULIZAPortalIE(InfoExtractor):
+    IE_DESC = 'ulizaportal.jp - PIA LIVE STREAM'
+    _VALID_URL = r'https?://ulizaportal\.jp/pages/(?P<id>[\da-f]{8}-[\da-f]{4}-[\da-f]{4}-[\da-f]{4}-[\da-f]{12})'
+    _TESTS = [{
+        'url': 'https://ulizaportal.jp/pages/005f18b7-e810-5618-cb82-0987c5755d44',
+        'info_dict': {
+            'id': '005f18b7-e810-5618-cb82-0987c5755d44',
+            'title': 'プレゼンテーションプレイヤーのサンプル',
+            'live_status': 'not_live',
+        },
+        'params': {
+            'skip_download': True,
+            'ignore_no_formats_error': True,
+        },
+    }, {
+        'url': 'https://ulizaportal.jp/pages/005e1b23-fe93-5780-19a0-98e917cc4b7d?expires=4102412400&signature=f422a993b683e1068f946caf406d211c17d1ef17da8bef3df4a519502155aa91&version=1',
+        'info_dict': {
+            'id': '005e1b23-fe93-5780-19a0-98e917cc4b7d',
+            'title': '【確認用】視聴サンプルページ（ULIZA）',
+            'live_status': 'not_live',
```

Comment:
No m3u8 url can be extracted for this test. Did it expire? timestamp says it's good til the year 2099

Reply:
It works on my side:

```
[info] Available formats for 005e1b23-fe93-5780-19a0-98e917cc4b7d:
ID   EXT RESOLUTION │   TBR PROTO │ VCODEC  ACODEC
───────────────────────────────────────────────────
1200 mp4 unknown    │ 1200k m3u8  │ unknown unknown
2300 mp4 unknown    │ 2300k m3u8  │ unknown unknown
4600 mp4 unknown    │ 4600k m3u8  │ unknown unknown
[debug] Default format spec: bestvideo*+bestaudio/best 
[info] 005e1b23-fe93-5780-19a0-98e917cc4b7d: Downloading 1 format(s): 4600 
[info] There are no video thumbnails to download 
[info] Writing video metadata as JSON to: 【確認用】視聴サンプルページ（ULIZA） [005e1b23-fe93-5780-19a0-98e917cc4b7d].info.json
[debug] Invoking hlsnative downloader on "https://d3b2nkk5uuz3xf.cloudfront.net/5165744460395/20210114121519_7aa7c28d5006907933df1d4a495206decd8dc14bf808af21c757a7dc16aa9999_1.m3u8"
[hlsnative] Downloading m3u8 manifest 
[hlsnative] Total fragments: 6 
[download] Destination: 【確認用】視聴サンプルページ（ULIZA） [005e1b23-fe93-5780-19a0-98e917cc4b7d].mp4 
[debug] File locking is not supported. Proceeding without locking
[download]  38.5% of ~  31.65MiB at  686.85KiB/s ETA 00:29 (frag 2/6)

```

Reply:
Ah then I must be geo-blocked